server/src/test/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdSettingsTests.java

```diff
@@ -487,35 +801,6 @@ private void doTestDescriptions(boolean includeKey) {
             equalTo(frozenFloodWatermarkPrefix + "91.5%")
         );
 
-        diskThresholdSettings = new DiskThresholdSettings(
```

Comment:
I am not sure I follow why this absolute specification case no longer needs testing?

Reply:
Good catch! I think this got removed by mistake when I was trying to merge master/main after the sub-PRs I had merged. I re-instated this part of the test. Thanks!